src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -397,9 +406,14 @@ public enum PowerKeepers implements LogicCardInfo {
             checkNoSPC()
             checkLastTurn()
             assert my.hand.filterByType(ENERGY) : "No Energy in hand"
-            my.hand.filterByType(ENERGY).select("Discard").discard()
-            draw choose([0,1,2,3], ["0","1","2","3"], "Draw how many cards?", 3)
+            //TODO: Confirm whether to use EX or LV.X ruling regarding being able to use it without cards in deck (EX ruling: You can't use the power at all; LV.X ruling: you can, you discard the energy and nothing else happens)
```

Comment:
I'd say newest ruling trumps an older ruling especially if they are about the same exact situation and card.

Reply:
Yup, will remove this TODO in #877 after checking around and talking with players at the time. Specially with the card itself being the specific one the ruling was altered on shortly after.